### DISCUSSION TITLE
Extract all edge_id informations

### DISCUSSION THREAD

**Discussion by mushabtinumbang** [gis-ops/docker-valhalla#155] [Q&A] [answered]
Hi, is it possible to get all edge_ids information that valhalla has generated? For every edge_id's generated, i would like to extract the geometry (longlat) and the particular osm_way_ids. I am still new in Valhalla, would appreciate if anyone can help on this one. Thanks!

**Comment by nilsnolde** [comment, marked as answer]
https://valhalla.github.io/valhalla/api/map-matching/api-reference/#trace-attributes-action

unless it's about docker setup stuff, best to ask at https://github.com/valhalla/valhalla

**Reply by mushabtinumbang** [reply]
Thanks for answering, I will proceed to ask another question on https://github.com/valhalla/valhalla
